webdev3.html

```diff
@@ -0,0 +1,194 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<title>Greatest Leaflet Map Ever</title>
+   <!-- External Stylesheets -->
+    <link rel="stylesheet" href="https://unpkg.com/leaflet@1.5.1/dist/leaflet.css" />
+      <style>
+           
+/* Allow Map to fill page*/
+/* source: http://bl.ocks.org/d3noob/7654694 */
+/* CSS adjusted to display map coordinates source: https://kristofj-umd.github.io/webdev-f19/assignments/webdev3.html*/
+        body {
+            padding: 0;
+            margin: 0;
+                }
+        html, body, #map {
+            height: calc(100vh - 60px);
```

Comment:
Interesting solution to the map height. I wonder if this is why your zoom control is over the top of your header.